host_vars/dalite.harvardx.harvard.edu/public.yml

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
@antoviaque This was added here after our [discussion here](https://github.com/open-craft/deployment-deploy-services/pull/1#discussion_r60221179). Most of these variables are also role defaults, but I needed to move them to top-level scope so other roles (swift backup, sanity checker) has access to them. Previously I just hardcoded a couple of paths, which seemed to be lesser evil. 

When we have second dalite instance deployed from this repository, we might as well create a ``group_vars/dalite`` and it'll work. 

Of course I could cram all variables to private vars... but then: 

1. Value of this repository would be much smaller for another people 
2. Deploying new instances of dalite will be much less fun.